### PR TITLE
Profiling tool: Health check follow on

### PR DESCRIPTION
**Pull request by nartal1**
Signed-off-by: Niranjan Artal <nartal@nvidia.com>
This addresses review comments and nits from https://github.com/NVIDIA/spark-rapids/pull/2632#pullrequestreview-678544181

### CODE REVIEW

**Comment by nartal1** [comment]
build

File: tools/README.md [hunk 1]
```diff
@@ -339,25 +339,25 @@ Shuffle Skew Check: (When task's Shuffle Read Size > 3 * Avg Stage-level size)
...
-|4      |0             |2908  |0      |TaskKilled(another attempt succeeded|
-|4      |0             |3410  |1      |ExceptionFailure(ai.rapids.cudf.Cudf|
-+-------+--------------+------+-------+------------------------------------+
-
-Application application_1616746343401_0025 (index=1) failed stages:
+Failed tasks:
++-------+--------------+------+-------+----------------------+
+|stageId|stageAttemptId|taskId|attempt|endReason_first100char|
++-------+--------------+------+-------+----------------------+
+|      4|             0|  2842|      0|  ExceptionFailure(...|
```
Comment (tgravescs) [inline tools/README.md:346]: this doesn't look like 100 chars?

Comment (nartal1) [reply tools/README.md:346]: Yes, For readibility purpose had truncated the output.  I will update correct output in the README and we can decide if it's okay to keep it.

**Review by tgravescs — APPROVED** [review]

**Comment by tgravescs** [comment]
build

**Comment by nartal1** [comment]
build

**Pull request merged**